src/app/pages/Designathons/Designathon24/components/FAQ/FAQ.module.scss

```diff
@@ -114,6 +114,7 @@
 	top: 0;
 	right: 0;
 	max-width: 30%;
+	pointer-events: none;
```

Comment:
Thought: It appears these are all being applied to decorative images. Could we instead use another utility class on the `img` elements to avoid repeating this style in several class names? This would also improve flexibility in case we need to apply any other styles to such decorative images.